challenge_2/python/alexbotello/main.py

```diff
@@ -0,0 +1,6 @@
+if __name__ == '__main__':
+    N = int(input()) # How many items in the list?
+    array = [int(input()) for _ in range(N)] # Enter ints
+
+    print("\nThe single number is ", end='')
+    print(*[i for i in array if array.count(i) == 1], sep=' or ')
```

Comment:
This is really slow, it's O(N^2) because the array isn't sorted so the `count()` method has to iterate through every element in your array. It's essentially a doubly nested for loop.